### ISSUE TITLE
Does it support react-native-web?

### ISSUE THREAD

**Issue by im-fast** [flatlogic/react-native-starter#89]
Does it support react-native-web?

**Comment by pmartini** [comment]
> Does it support react-native-web?

yes, no way to run it on web. You can try to manually disable libraries that are not working on web, probably it will help
